hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/TestS3AAWSCredentialsProvider.java

```diff
@@ -18,8 +18,7 @@
 
 package org.apache.hadoop.fs.s3a;
 
-import java.io.IOException;
-import java.io.InterruptedIOException;
+import java.io.*;
```

Comment:
revert; .* is not allowed in the hadoop code as it is too brittle

Reply:
Fixed.